### DISCUSSION TITLE
Icon update: galaxy

### DISCUSSION THREAD

**Discussion by Arkellys** [FortAwesome/Font-Awesome#21269] [Feature Requests]
Except maybe for the duotone style, the current [galaxy](https://fontawesome.com/icons/galaxy?s=solid&f=classic) icon looks very much like a shuriken. I suppose it's a little tricky to represent a galaxy as an icon, but maybe it could be updated to be more twisted or tilted so it's recognizable? Here are some examples:

![image](https://user-images.githubusercontent.com/32762018/219345434-96f0437c-4a81-4f0f-a0f2-dcdf51e3f0c0.png)
Original sources: [[1](https://www.iconspng.com/image/35332/spiral-galaxy)] [[2](https://www.stocklib.fr/media-172991198/galaxy-logo-icon-design-vector-illustration.html?keyword=blackhole)] [[3](https://thenounproject.com/icon/galaxy-22802/)]

**Comment by tagliala** [comment]
Hi!

> looks very much like a shuriken

Strict but fair

Let's assign to @sensibleworld as an enhancement
